packages/framework/src/Resources/views/Admin/Form/theme.html.twig

```diff
@@ -66,14 +66,14 @@
         {% endif %}
     {% else %}
         {% if render_form_row %}
-            <div class="form-line{{ disabledField is defined ? ' form-input-disabled form-line--disabled' }}">
+            <div class="form-line{{ attr.disabledField is defined ? ' form-input-disabled form-line--disabled' }}">
             {{ form_errors(form) }}
             {{ form_label(form, label) }}
         {% endif %}
         {{ form_widget(form) }}
-        {% if disabledField is defined %}
-            <span class="form-disabled {{ disabledField.class }} js-tooltip" data-toggle="tooltip" data-placement="{{ disabledField.placement|default('right') }}"
-                  title="{{ disabledField.title }}">
+        {% if attr.disabledField is defined %}
+            <span class="form-disabled {{ attr.disabledFieldClass|default('') }} js-tooltip" data-toggle="tooltip" data-placement="{{ attr.disabledFieldPlacement|default('right') }}"
```

Comment:
`disabledFieldPlacement` is not defined anywhere, how come?

Reply:
i think it is ready to use feature.